sunnypilot/selfdrive/controls/lib/dynamic_experimental_controller.py

```diff
@@ -0,0 +1,376 @@
+#!/usr/bin/env python3
```

Comment:
We want to have all SP related things (to the extent possible) on `/sunnypilot/xxx`